linkerd/proxy/http/src/client.rs

```diff
@@ -67,141 +65,129 @@ pub enum ClientServiceFuture {
 
 // === impl Layer ===
 
-pub fn layer<T, B>(h2_settings: crate::h2::Settings) -> Layer<T, B>
+pub fn layer<B>(h2_settings: crate::h2::Settings) -> Layer<B>
 where
     B: hyper::body::Payload + Send + 'static,
 {
     Layer {
         h2_settings,
-        _p: PhantomData,
+        _marker: PhantomData,
     }
 }
 
-impl<T, B> Clone for Layer<T, B>
+impl<B> Clone for Layer<B>
 where
     B: hyper::body::Payload + Send + 'static,
 {
     fn clone(&self) -> Self {
         Self {
             h2_settings: self.h2_settings,
-            _p: PhantomData,
+            _marker: self._marker,
```

Comment:
out of curiosity: is there a reason that this is now "copied" instead of just initialized to `PhantomData`? AFAICT, this should be equivalent, so I just wanted to make sure I wasn't missing something.

Reply:
Nope. Since I realized that phantomdata was copy i've been doing this in general...